### DISCUSSION TITLE
Asynchronous Verification of sessions

### DISCUSSION THREAD

**Discussion by toby63** [element-hq/element-meta#694] [Ideas]
#### What would you like to do?

Similar to vector-im/element-web#10303 (but that one mentioned verification between users regarding E2E-Encryption).

I would like to be able to verify a new device or session, even if I don't have the other device directly available.

#### Why would you like to do it?

Because it is much more convenient this way.

#### How would you like to achieve it?

I think it should be possible to simply verify a device from an already verified device, without needing a confirmation from that other device.

### Have you considered any alternatives?

-

### Additional context

-

**Comment by kittykat** [comment]
Moving to discussions as this is cross-platform :+1:
